### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nayeon-upstage**
- Add cookbook description
- Add API List
- Add Cookbook List

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,65 @@
...
+  
+
+### API List
+
+| API | Description | Import | Example usage |
+| --- | --- | --- | --- |
+| Chat | Build assistants using Solar Mini Chat | from langchain_upstage import ChatUpstage | [Link](https://python.langchain.com/v0.1/docs/integrations/chat/upstage/) |
+| Text Embedding | Embed strings to vectors | from langchain_upstage import UpstageEmbeddings | [Link](https://python.langchain.com/v0.1/docs/integrations/text_embedding/upstage/) |
+| Groundedness Check | Verify groundedness of assistant's response | from langchain_upstage import UpstageGroundednessCheck | [Link](https://python.langchain.com/v0.1/docs/integrations/tools/upstage_groundedness_check/) |
+| Layout Analysis | Serialize documents with tables and figures | from langchain_upstage import UpstageLayoutAnalysisLoader | [Link](https://python.langchain.com/v0.1/docs/integrations/document_loaders/upstage/) |
```
Comment (chosh0615) [inline README.md:43]: Column 3, 4 are LangChain-specific. I suggest to remove them and add link to our developer's documentation.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,65 @@
...
+| Solar-LLM-ZeroToAll | [12_summary_writing_translation.ipynb](https://github.com/UpstageAI/cookbook/blob/main/Solar-LLM-ZeroToAll/12_summary_writing_translation.ipynb) | This notebook shows using the LangChain library with the Upstage Embeddings model for various natural language processing tasks such as summarization, text simplification, and translation. It showcases how to create prompts and chains for summarizing text, rewriting text to be more understandable, translating English to Korean, and providing translations in a specific style by conditioning on examples. |
+|  | [langgraph_self_rag.ipynb](https://github.com/UpstageAI/cookbook/blob/main/LangGraph-Self-RAG/langgraph_self_rag.ipynb) | This notebook provides a comprehensive guide on implementing Self-Retrieval-Augmented Generation (Self-RAG) using LangGraph. It demonstrates how to enhance text generation by integrating self-retrieval mechanisms, allowing the model to dynamically fetch relevant information to improve accuracy and relevance.  |
```
Comment (chosh0615) [inline README.md:65]:
```suggestion
| RAG | [langgraph_self_rag.ipynb](https://github.com/UpstageAI/cookbook/blob/main/LangGraph-Self-RAG/langgraph_self_rag.ipynb) | This notebook provides a comprehensive guide on implementing Self Retrieval-Augmented-Generation (Self-RAG) using LangGraph. It demonstrates how to enhance text generation by integrating self-retrieval mechanisms, allowing the model to dynamically fetch relevant information to improve accuracy and relevance.  |
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,65 @@
...
+
+**Content:**
+
+- Source code for apps, programs, or computational projects assisted by the Upstage APIs.
+- Tutorials, educational materials, or lesson plans developed with the Upstage APIs.
+- Any other interesting applications that showcase the Upstage API's potential!
+
+**Requirements:**
+
+- Code/text entries should be in .md, .ipynb or other common formats
```
Comment (chosh0615) [inline README.md:20]:
```suggestion
- Code/text entries should be in .md, .ipynb or other common formats.
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,65 @@
...
+- Include an overview describing the project/output and its purpose. Provide step-by-step instructions in a logical order and Include any relevant tips, notes, or variations.
+- Projects should be original creations or properly credited if adapted from another source.
+- Submissions should not contain explicit violating content
```
Comment (chosh0615) [inline README.md:23]:
```suggestion
- Submissions should not contain explicit violating content.
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,65 @@
...
+
+**Rights:**
+
+- You maintain ownership and rights over your submitted project
+- By submitting, you grant us permission to showcase and distribute the entry
+- Entries cannot infringe on any third-party copyrights or intellectual property
```
Comment (chosh0615) [inline README.md:29]:
```suggestion
- You maintain ownership and rights over your submitted project.
- By submitting, you grant us permission to showcase and distribute the entry.
- Entries cannot infringe on any third-party copyrights or intellectual property.
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,65 @@
...
+| Solar-LLM-ZeroToAll | [05_PDF_CAG.ipynb](https://github.com/UpstageAI/cookbook/blob/main/Solar-LLM-ZeroToAll/05_PDF_CAG.ipynb) | Check how to analyze PDF documents using Solar. It includes steps to extract text from PDFs, process the text using various natural language processing techniques, and generate comprehensive analyses.  |
+| Solar-LLM-ZeroToAll | [06_LA_CAG.ipynb](https://github.com/UpstageAI/cookbook/blob/main/Solar-LLM-ZeroToAll/06_LA_CAG.ipynb) | Explore how to perform layout analysis on documents using a Solar. It includes techniques for detecting and interpreting the structural elements of a document, such as headings, paragraphs, tables, and figures. And chat based on result. |
+| Solar-LLM-ZeroToAll | [07_RAG.ipynb](https://github.com/UpstageAI/cookbook/blob/main/Solar-LLM-ZeroToAll/07_RAG.ipynb) | Learn how to use of Retrieval Augmented Generation (RAG). It showcases how to load documents, create a retriever, and use it with an Solar to answer questions based on retrieved relevant information. |
```
Comment (chosh0615) [inline README.md:56]:
```suggestion
| Solar-LLM-ZeroToAll | [07_RAG.ipynb](https://github.com/UpstageAI/cookbook/blob/main/Solar-LLM-ZeroToAll/07_RAG.ipynb) | Learn how to implement Retrieval Augmented Generation (RAG). It showcases how to load documents, create a retriever, and use it with an Solar to answer questions based on retrieved relevant information. |
```

**Review by JuHyung-Son — APPROVED** [review]
오우 많네요. 수고하셨습니다!

**Pull request merged**